Sources/Instructions/Extra/Default Views/CoachMarkBodyDefaultView.swift

```diff
@@ -40,11 +42,15 @@ public class CoachMarkBodyDefaultView: UIControl,
 
     // MARK: - Initialization
     override public init(frame: CGRect) {
+        self.nextLabelPosition = .none
+
         super.init(frame: frame)
         initializeViewHierarchy()
     }
 
-    public init(frame: CGRect, hintText: String, nextText: String?) {
+    public init(frame: CGRect, hintText: String, nextText: String?, nextLabelPosition: CoachMarkNextLabelPosition?) {
```

Comment:
We probably want to force users to provide a position that isn't nil.

```suggestion
    public init(frame: CGRect, hintText: String, nextText: String?, nextLabelPosition: CoachMarkNextLabelPosition) {
```